tests/CatalogMetadataTests/AzureStorageFacts.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using Xunit;
+using NuGet.Services.Metadata.Catalog.Persistence;
+
+namespace CatalogMetadataTests
+{
+    public class AzureStorageFacts : AzureStorageBaseFacts
+    {
+        public AzureStorageFacts() : base()
+        {
+        }
+
+        [Theory]
+        [InlineData(true, true, "SHA512Value1", true, true, "SHA512Value1", true)]
+        [InlineData(true, true, "SHA512Value1", true, true, "SHA512Value2", false)]
+        [InlineData(false, false, null, true, true, "SHA512Value1", true)]
+        [InlineData(true, true, "SHA512Value1", false, false, null, false)]
+        [InlineData(false, false, null, false, false, null, true)]
+        [InlineData(true, false, null, true, true, "SHA512Value1", false)]
+        [InlineData(true, true, "SHA512Value1", true, false, null, false)]
+        [InlineData(true, false, null, true, false, null, false)]
+        public async void ValidateAreSynchronizedmethod(bool isSourceBlobExisted,
```

Comment:
>isSourceBlobExisted [](start = 61, length = 19)

What do you think of the name `sourceBlobExists`? Same comment for `isDestinationBlobExisted`

Reply:
Ok